### DISCUSSION TITLE
データベースとアプリケーションの通信で、1回あたりどれくらいのデータ容量が送れるのか？

### DISCUSSION THREAD

**Discussion by yuyakawai0411** [kakeru-one/web-engineering-roundtable#41] [Q&A] [answered]
## 背景
データベースとアプリケーションは、TCP/IPプロトコルで通信していると思われる。
1パケットあたり、どれくらいのデータサイズが送れるか? そのサイズは変更することができるのか気になった
https://repost.aws/ja/knowledge-center/rds-mysql-server-gone-away

**Comment by yuyakawai0411** [comment]
以下のサイトでDBからアプリケーションに返すデータサイズは小さい方がいいと言っている

https://docs.aws.amazon.com/ja_jp/AmazonRDS/latest/AuroraUserGuide/apg-waits.clientwrite.html

1パケットあたり、どれくらいのデータサイズが送れるか?はMTUと呼ばれており、ジャンボフレームを使うとMTUは増加する。
https://zenn.dev/tsumita7/articles/are-you-aware-of-mtu-in-aws

**Reply by kakeru-one** [reply, marked as answer]
`MTU = 1500バイト`
デフォルトでは上記の値で、ジャンボフレームを使うとMTUは増加させることができる。
しかし、データベースから送信するデータのデータサイズを小さくするのが定石なので、先にこちらを解消すること。
